homeassistant/components/vicare/climate.py

```diff
@@ -230,7 +230,10 @@ def update(self) -> None:
     @property
     def hvac_mode(self) -> HVACMode | None:
         """Return current hvac mode."""
-        return VICARE_TO_HA_HVAC_HEATING.get(self._current_mode)
+        if self._current_mode is None:
+            return None
+        mode: str = self._current_mode
```

Comment:
You can solve this by doing `self._current_mode: str | None = None` in the constructor (you can take it out of the constructor then). You can remove this line after and directly use it in the `.get()`